public/index.html

```diff
@@ -6,8 +6,169 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Untitled</title>
         <link rel="author" href="humans.txt">
+        <link> <link rel="stylesheet" type="text/css" href="facesmashmain.css">
+    
+    <title>"Facesmash"</title>          
+    
     </head>
+    
     <body>
-        <h1>Audacious Aretes</h1>
+        <ul class="fixedheader clearfix">
+            <li><h1><a href = "/">Facesmash</a></h1></li>
+                <li>
+                <ul>
+                    <li><a href ="/">About Me</a></li>
+                    <li><a href ="/">Sign Up</a></li>
+                    <li><a href ="/">Login</a></li>
+                </ul>
+            </li>
+        </ul>
+        </ul>
+        <form>
+        
+            <div id="main">
+            
+            <p> Connect with your friends and all of the other billions of people in the world that you don't even know. </p>
```

Comment:
Okay I will stop doing that. It helps me read it better and just lets me make sure it's in right spots spacing helps me see better but if it's not right I will fix it